src/internal/util.ts

```diff
@@ -1,9 +1,11 @@
 import fc, { Arbitrary } from 'fast-check'
 
+/* istanbul ignore next */
 /** @internal */
 export const base64Encode = (s: string): string =>
   Buffer ? Buffer.from(s).toString('base64') : btoa(s)
 
+/* istanbul ignore next */
```

Comment:
I thought I covered all branch cases, but `jest` disagreed,  so instead I used `istanbul ignore next`

Reply:
Oops. The default case is unreachable and Istanbul doesn't know it

Reply:
Oh I see, that makes sense.  So we should be able to remove the explicit final case, and remove the istanbul-ignore-next?